modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -5,89 +5,108 @@
 [id="cluster-maximums-major-releases_{context}"]
 = {product-title} tested cluster maximums for major releases
 
-Tested Cloud Platforms for {product-title} 3.x: {rh-openstack-first}, Amazon Web Services and Microsoft Azure.
-Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft Azure and Google Cloud Platform.
+[NOTE]
+====
+Red Hat does not provide direct guidance on sizing your {product-title} cluster. This is because determining whether your cluster is within the supported bounds of {product-title} requires careful consideration of all the multidimensional factors that limit the cluster scale.
+====
+
+{product-title} supports tested cluster maximums rather than absolute cluster maximums. Not every combination of {product-title} version, control plane workload, and network plugin are tested, so the following table does not represent an absolute expectation of scale for all deployments. It might not be possible to scale to a maximum on all dimensions simultaneously. The table contains tested maximums for specific workload and deployment configurations, and serves as a scale guide as to what can be expected with similar deployments.
 
-[options="header",cols="3*"]
+[options="header",cols="2*"]
 |===
-| Maximum type |3.x tested maximum |4.x tested maximum
+| Maximum type |4.x tested maximum
 
 | Number of nodes
-| 2,000
 | 2,000 ^[1]^
 
 | Number of pods ^[2]^
 | 150,000
-| 150,000
 
 | Number of pods per node
-| 250
-| 500 ^[3]^
+| 2,500 ^[3]^
```

Comment:
My understanding is that this result is experimental and we probably don't want to document it as a tested maximum.  I suggest leaving it at 500 and publishing the 2500 number in a blog or the like.   @afcollins, @rsevilla87 , @jtaleric fyi if you have further comments on this.

Reply:
Agree, 2.500 ppn has not been intensively tested w/ complex workloads

Reply:
As discussed, I will keep this as 500 to be safe. If we want to point to a blog post later, we can do that in a separate PR. Thank you!